core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -7072,6 +7144,175 @@ components:
+      properties:
+        network:
+          $ref: "#/components/schemas/NetworkIdentifier"
+        filter:
+          $ref: "#/components/schemas/StreamProofsFilter"
+        max_page_size:
+          description: If specified, the maximum number of proofs that will be returned.
+          type: integer
+        continuation_token:
+          $ref: "#/components/schemas/ContinuationToken"
+    StreamProofsFilterType:
+      type: string
+      enum:
+        - All
+        - NewEpochs
+        - ProtocolUpdateTriggers
+        - ProtocolUpdateExecution
+    StreamProofsFilter:
+      type: object
+      required:
+        - type
+      properties:
+        type:
+          $ref: "#/components/schemas/StreamProofsFilterType"
+      discriminator:
+        propertyName: type
+        mapping:
+          # NOTE: These need to match StreamProofsFilterType
+          All: '#/components/schemas/StreamProofsFilterAll'
+          NewEpochs: '#/components/schemas/StreamProofsFilterNewEpochs'
+          ProtocolUpdateInitializations: '#/components/schemas/StreamProofsFilterProtocolUpdateInitializations'
+          ProtocolUpdateExecution: '#/components/schemas/StreamProofsFilterProtocolUpdateExecution'
+    StreamProofsFilterAll:
+      allOf:
+        - $ref: "#/components/schemas/StreamProofsFilter"
+        - type: object
+          properties:
+            from_state_version:
+              $ref: "#/components/schemas/StateVersion"
+              description:
+                The first proof to be returned should be at this state version or above.
+                If empty, it starts from 0.
+    StreamProofsFilterNewEpochs:
+      allOf:
+        - $ref: "#/components/schemas/StreamProofsFilter"
+        - type: object
+          properties:
+            from_epoch:
+              type: integer
+              format: int64
+              minimum: 0
+              maximum: 10000000000
+              description:
+                The first proof to be returned should be the proof starting this epoch.
+                If empty, it starts from the first epoch proof after genesis.
+                The network status endpoint can be used to find the current epoch.
+    StreamProofsFilterProtocolUpdateInitializations:
+      allOf:
+        - $ref: "#/components/schemas/StreamProofsFilter"
+        - type: object
+          properties:
+            from_state_version:
+              $ref: "#/components/schemas/StateVersion"
+              description:
+                The first proof to be returned should be at this state version or above.
+                If empty, it starts from 0.
+    StreamProofsFilterProtocolUpdateExecution:
+      allOf:
+        - $ref: "#/components/schemas/StreamProofsFilter"
+        - type: object
+          properties:
+            protocol_version:
+              type: string
+              description: |
+                The protocol version name to filter to. This can be returned from looking at the protocol updates.
```

Comment:
I don't understand the "... returned from looking at ..." part.

Reply:
Will clarify, thanks 👍